AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participants-list/participants-list.component.ts

```diff
@@ -64,4 +65,34 @@ export class ParticipantsListComponent implements OnInit {
     get selectedParticipantToRemove() {
         return this.$selectedForRemove;
     }
+
+    isInterpreter(participant: ParticipantModel): boolean {
+        return participant.hearing_role_name.toLowerCase().trim() === HearingRoles.INTERPRETER.toLowerCase();
+    }
+    isRepresentative(participant: ParticipantModel): boolean {
+        return participant.hearing_role_name.toLowerCase().trim() === HearingRoles.REPRESENTATIVE.toLowerCase();
+    }
+    getInterpreteeDisplayName(participant: ParticipantModel): string {
+        let interpretee: ParticipantModel;
+        if (
+            this.isEditMode &&
+            participant.hearing_role_name.toLowerCase().trim() === HearingRoles.INTERPRETER &&
+            participant.linked_participants.length > 0
+        ) {
+            interpretee = this.participants.find(p => p.id === participant.linked_participants[0].linkedParticipantId);
+        } else {
+            interpretee = this.participants.find(p => p.email === participant.interpreterFor);
+        }
+        const interpretedForName = interpretee ? interpretee.display_name : '';
+        return interpretedForName;
+    }
+    isInterpretee(participant: ParticipantModel): boolean {
+        let interpretee: ParticipantModel;
+        if (this.isEditMode && participant.linked_participants && participant.linked_participants.length > 0) {
+            interpretee = this.participants.find(p => p.id === participant.linked_participants[0].linkedParticipantId);
+        } else {
+            interpretee = this.participants.find(p => p.interpreterFor === participant.email);
+        }
+        return interpretee ? true : false;
```

Comment:
As above, consider `some` then just return `interpretee`

Reply:
update to use some()